code/game/objects/items/weapons/shields.dm

```diff
@@ -41,13 +41,13 @@
 	. = ..()
 	switch(get_block_chance(user))
 		if(0 to 30)
-			to_chat(user, "So heavy... You feel doubtful in your ability to parry with this shield. Maybe if you changed your grip?")
```

Comment:
you can't get this low without holding a riot shield the wrong way, hence the hint towards changing your grip

quite intentional

Reply:
This is the examine text you get on any riot shield when you aren't walking. Even with 200 rob, a riot shield shows this examine text.

Changing grip didn't tell me, as a new player, that ROB was a governing stat, nor that I should walk with riot shields.

Reply:
> Changing grip didn't tell me, as a new player, that ROB was a governing stat, nor that I should walk with riot shields.

Then change it to tell you more clearly. What you wrote is just misleading, rather than confusing.